src/main/kotlin/uk/gov/dluhc/notificationsapi/rest/StatisticsController.kt

```diff
@@ -16,11 +18,10 @@ import uk.gov.dluhc.notificationsapi.service.SentNotificationsService
 class StatisticsController(
```

Comment:
After making these changes I started questioning whether I should have split out the endpoints (as I have done), or used one endpoint and had signatureRequested and photoRequested as optional fields in one return type (rather than the two separate return types for VAC and OAVA).

Benefit of the latter is would probably require slightly fewer changes for VAC (particularly if we kept the endpoint the same and sent the sourceType in query parameters rather than path), but it would still require changes, and having an optional 'signatureRequested' field for a service where signatures aren't required could be a bit strange. So have kept as is for now but interested to hear other thoughts!

Reply:
I think splitting the endpoints to account for the different return types makes sense :)